Rubberduck.sln

```diff
@@ -95,7 +95,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Rubberduck.Interaction", "R
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Rubberduck.UnitTesting", "Rubberduck.UnitTesting\Rubberduck.UnitTesting.csproj", "{68F55404-32CA-4414-96B8-E284F3B1F846}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Rubberduck.JunkDrawer", "Rubberduck.JunkDrawer\Rubberduck.JunkDrawer.csproj", "{D7A3F7A3-BD90-4845-B098-AC3AFBF0E16F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Rubberduck.JunkDrawer", "Rubberduck.JunkDrawer\Rubberduck.JunkDrawer.csproj", "{D7A3F7A3-BD90-4845-B098-AC3AFBF0E16F}"
```

Comment:
Interesting...?

Reply:
Ugh. They really had to use a very lame format for `.sln` format. So that guid is a project type. [According to this page](https://www.codeproject.com/reference/720512/list-of-visual-studio-project-type-guids) the "FAE..." guid means it's a `Windows C#'. But `9A1` isn't listed there.... [Nor is it listed on MZ Tools](https://www.mztools.com/articles/2008/MZ2008017.aspx).... 

[Apparently it's not even officially documented](https://social.msdn.microsoft.com/Forums/en-US/609a3d1e-9901-43b9-9b5e-da8bda318a8d/new-project-guids). Argh!